<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSNavigationStepTumourSelection</class>
 <widget class="QWidget" name="QmitkUSNavigationStepTumourSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>First the body marker has to be placed on the surface. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="bodyMarkerTrackingStatusLabel">
     <property name="styleSheet">
      <string notr="true">
              background-color: #ff7878; margin-right: 1em; margin-left: 1em;
              border: 1px solid grey
            </string>
     </property>
     <property name="text">
      <string>Body marker is not inside the tracking volume.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="tumourSearchExplanationLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Search the tumour by using the ultrasound modality. Freeze imaging, when the best position is reached.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkUSNavigationFreezeButton" name="freezeImageButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tumourSelectionExplanation1Label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Left click in the center of the tumour on the ultrasound image.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tumourSelectionExplanation2Label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Move the mouse pointer for changing the size of the circle to match the arc radius of the tumour. Another mouse click will fix the size. Right clicking will abort the process and will let you place the center point againg.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="tumourSizeExplanationLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can move on with the next step now. In case the circle does not match the tumour you can change the arc radius by using the Tumour Size slider. Tumour marking can be started again by deleting the tunour from the image.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tumourSizeLayout">
     <item>
      <widget class="QLabel" name="tumourSizeLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Tumour Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="tumourSizeSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="deleteTumourButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete Tumour From Image</string>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/USNavigation.qrc">
       <normaloff>:/USNavigation/minus.png</normaloff>:/USNavigation/minus.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkUSNavigationFreezeButton</class>
   <extends>QPushButton</extends>
   <header>src/internal/Widgets/QmitkUSNavigationFreezeButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources/USNavigation.qrc"/>
 </resources>
 <connections/>
</ui>
